terraform-plans/templates/github/snap_tics.yaml.tftpl

```diff
@@ -0,0 +1,68 @@
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
+name: TICS
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Verify snap builds successfully
+        id: build
+        uses: canonical/action-build@v1
+
+      - name: Upload the built snap
+        uses: actions/upload-artifact@v4
+        with:
+          name: snap_artifact
```

Comment:
Why do you have a build job here? There is already a build check in the check workflow. If we do really need to use a newly built snap here, then please avoid uploading artefacts - simply build the snap and run the tics target in the same job. :)

Reply:
(unless of course there's a need to retain the built snap artefact for some other purpose)